cvat/apps/engine/tests/test_rest_api.py

```diff
@@ -4562,6 +4612,47 @@ def _send_data_and_fail(*args, **kwargs):
                     image_sizes, StorageMethodChoice.FILE_SYSTEM, StorageChoice.LOCAL,
                     send_data_callback=_send_data_and_fail)
 
+    def _test_api_v2_tasks_id_data_create_can_use_server_rar(self, user):
+        task_spec = {
+            "name": 'task rar in the shared folder #32',
+            "overlap": 0,
+            "segment_size": 0,
+            "labels": [
+                {"name": "car"},
+                {"name": "person"},
+            ]
+        }
+
+        task_data = {
+            "server_files[0]": "test_rar.rar",
+            "image_quality": 75,
+            "copy_data": False,
+            "use_cache": True,
+        }
+        image_sizes = self._share_image_sizes[task_data["server_files[0]"]]
+
+        self._test_api_v2_tasks_id_data_spec(user, task_spec, task_data, self.ChunkType.IMAGESET, self.ChunkType.IMAGESET,
+            image_sizes, StorageMethodChoice.CACHE, StorageChoice.LOCAL)
+
+    def _test_api_v2_tasks_id_data_create_can_use_local_rar(self, user):
```

Comment:
Shouldn't this be called from somewhere?

Reply:
Its called automatically because of the method name specification [here](https://github.com/cvat-ai/cvat/blob/develop/cvat/apps/engine/tests/test_rest_api.py#L4572)

Reply:
Huh. Alright.